doc/btp_cap.txt

```diff
@@ -0,0 +1,269 @@
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Discover and Subscribe
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+		Response parameters:	<none>
+
+		This command is used to discover all remote PACS, ASCS and BASS
+		characteristics. In case of an error, the error status response
+		will be returned. In case of a success, the IUT continues processing
+		the command asynchronously. During discovery, the IUT may send events:
+					Discover and Subscribe Completed event
+					Codec Capabilities Found event
+					ASE Found event
+
+	Opcode 0x03 - Ordered Access
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+		Response parameters:	<none>
+
+		This command is used to execute the CSIP Ordered Access.
+		In case of an error, the error status response will be returned.
+		In case of a success, the IUT continues processing
+		the command asynchronously.
+
+	Opcode 0x04 - Unicast Setup Stream
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					ASE_ID (1 octet)
+					CIS_ID (1 octet)
+					CIG_ID (1 octet)
+					Coding Format (1 octet)
+					VID (2 octets)
+					CID (2 octets)
+					Sdu Interval (3 octets)
+					Framing (1 octet)
+					Max SDU (2 octets)
+					Retransmission Num (1 octet)
+					Max Transport Latency (2 octets)
+					Presentation Delay (3 octets)
+					Codec Config LTVs length (1 octet)
+					Metadata LTVs length (1 octet)
+					LTVs (varies)
+
+		Response parameters:	<none>
+
+		This command is used to configure CISes locally before executing
+		the Unicast Audio Start procedure.
+		In case of an error, the error status response will be returned.
+
+	Opcode 0x05 - Unicast Audio Start
+		Controller Index:	<controller id>
+		Command parameters: CIG_ID (1 octet)
+					Set Type (1 octet)
+
+		Valid Set Type values:
+					0x00 = Ad-Hoc Set
+					0x01 = CSIP Coordinated Set
+
+		Response parameters: <none>
+
+		This command is used to execute the Unicast Audio Start procedure.
+		In case of an error, the error status response will be returned.
+		In case of a success, the IUT continues processing the command
+		asynchronously.
+
+	Opcode 0x06 - Unicast Audio Update
+		Controller Index:	<controller id>
+		Command parameters: Stream Count (1 octet)
+					Update Data (varies)
+
+		where Update Data can contain one or multiple structures defined as:
+					Address_Type (1 octet)
+					Address (6 octets)
+					ASE_ID (1 octet)
```

Comment:
Is it allowed? The CAP spec mentions `7.3.1.6.1 Updating Context Type and CCID Value Metadata` only

Reply:
I think `Update Data` is misleading. The address and ASE_ID is used only for selection of the right instance.